.github/workflows/flatpak.yaml

```diff
@@ -0,0 +1,29 @@
+on: 
+  push:
+    tags:
+      - '*'
+
+jobs: 
+  flatpak:
+     
+    runs-on: ubuntu-20.04
+    
+    steps:
+    - uses: actions/checkout@v2 
+    - name: Install Flatpak
+      run: >
+        sudo apt install flatpak gnome-software-plugin-flatpak && flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
Is `gnome-software-plugin-flatpak` required?

Reply:
You are right, that is only needed for updating flatpaks through a GUI instead of manually through a terminal.  It isn't relevant for this manifest unless it gets submitted to flatpak for general use, so I removed it just now.